src/_pytest/outcomes.py

```diff
@@ -89,6 +90,9 @@ def skip(msg="", **kwargs):
         It is better to use the :ref:`pytest.mark.skipif ref` marker when possible to declare a test to be
         skipped under certain conditions like mismatching platforms or
         dependencies.
+        Similarly, use ``# doctest: +SKIP`` directive (see `doctest.SKIP
+        <https://docs.python.org/3/library/doctest.html#doctest.SKIP>`_)
+        to skip a doctest statically.
```

Comment:
How does this work?
Can you add a test for it also, please?

Reply:
It's already tested; e.g.,

https://github.com/pytest-dev/pytest/blob/158432217c8ae7fdaf4e9c47156c72b8674d9fa5/testing/test_doctest.py#L858-L868

It is not the feature I added in this PR.  I think the SKIP directive has been working since forever.  I just added it has a "friendly pointer" as I mentioned doctest in the docstring for `pytest.skip`.